### PR TITLE
Updated French translations

### PR DESCRIPTION
**Pull request by biodranik**
Our user has contributed French translations. Please review.

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -14144,7 +14146,7 @@
     eu = 1. Ireki ezarpenak
     fa = 1. بازکردن تنظیمات
     fi = 1. Avaa asetukset
-    fr = 1. Ouvrir Paramètres
+    fr = 1. Ouvrir les Paramètres
```
Comment (Jean-BaptisteC) [inline data/strings/strings.txt:14149]:
```suggestion
    fr = 1. Ouvrir les paramètres
```

File: data/strings/strings.txt [hunk 2]
```diff
@@ -15390,6 +15394,7 @@
     de = Geben Sie eine gültige LINE-Webadresse oder LINE ID ein
     es = Introduzca una dirección web o ID de LINE válidos
     et = Sisesta kehtiv LINE veebiaadress või LINE ID
+    fr = Entrez une adresse web ou une ID de LIGNE valide
```
Comment (Jean-BaptisteC) [inline data/strings/strings.txt:15397]:
```suggestion
    fr = Entrez une adresse web ou un ID de LIGNE valide
```

File: data/strings/strings.txt [hunk 3]
```diff
@@ -24676,7 +24685,7 @@
     eu = Gaituta dagoenean, pantailak lo egin ahal izango du jarduerarik gabe egon ondoren.
     fa = درصورت فعال بودن ، بعد از یک دوره عدم فعالیت به صفحه اجازه خواب داده می شود.
     fi = Kun tämä asetus on käytössä, näyttö menee lepotilaan käyttämättömyyden jälkeen.
-    fr = Lorsqu'il est activé, l'écran sera autorisé à s'éteindre après une période d'inactivité.
+    fr = Lorsque ce paramêtre est activé, l'écran se mettra en veille après une période d'inactivité.
```
Comment (Jean-BaptisteC) [inline data/strings/strings.txt:24688]:
```suggestion
    fr = Lorsque ce paramètre est activé, l'écran se mettra en veille après une période d'inactivité.
```

**Review by Jean-BaptisteC — APPROVED** [review]
Some changes :)

**Comment by biodranik** [comment]
@Jean-BaptisteC Thank you! Fixed, PTAL
CC @vng

File: data/strings/strings.txt [hunk 4]
```diff
@@ -12888,7 +12890,7 @@
     eu = Negozioa gehitu
     fa = اضافه کردن یک تجارت
     fi = Lisää organisaatio
-    fr = Ajouter une organisation
+    fr = Ajouter une entreprise
```
Comment (Kazing) [inline data/strings/strings.txt:12893]: Not sure that "entreprise"  is the best word here, organisation maybe better
I don't see where this string will appear in the app, what is the context?

File: data/strings/strings.txt (hunk 2 above)
Comment (Kazing) [reply data/strings/strings.txt:15397]: fr = Entrez une adresse web ou un ID de LINE valide

LINE is the name of the app no need to translate it

**Review by Kazing — CHANGES_REQUESTED** [review]
hello, I have two comments to adress

File: data/strings/strings.txt (hunk 4 above)
Comment (Jean-BaptisteC) [reply data/strings/strings.txt:12893]: This translation is used on place page when you select a building, to create a new POI

Comment (Kazing) [reply data/strings/strings.txt:12893]: Ok I found it 👍
when i press on a building there is the string [placepage_add_business_button] en = Add business
when i long press anywhere else there is the string [placepage_add_place_button] en = Add a place to the map

both seems to open the editor in the same way, I don't see why we need two different strings, and moreover a building not necessarily contain a business (school, amenity, association etc...)

**Pull request merged**